.github/workflows/run-pr-tests.yaml

```
@@ -42,28 +42,105 @@ jobs:
+          output=true
+          for i in {0..${{steps.prep.outputs.pull_tries}}}
+          do
+            date=$(date -d "${i} days ago" +'%Y%m%d')
+            builder_tag="quay.io/${{ steps.prep.outputs.buildertags }}${date}"
+            docker pull ${builder_tag} || continue
+            echo "Successfully pulled ${builder_tag} from quay.io"
+            docker tag ${builder_tag} noobaa-builder
+            output=false
+            break
+          done
+          echo "should_build=${output}" >> $GITHUB_OUTPUT 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)